include/fmt/format.h

```diff
@@ -2896,7 +2896,7 @@ class bigint {
     auto size = other.bigits_.size();
     bigits_.resize(size);
     auto data = other.bigits_.data();
-    std::copy(data, data + size, bigits_.data());
+    std::uninitialized_copy(data, data + size, bigits_.data());
```

Comment:
Looks like `uninitialized_copy` is not `constexpr`. Let's replace it with

```c++
copy_str<bigit>(data, data + size, bigits_.data());
```

Reply:
Done!